### ISSUE TITLE
"Sign in with PKCE" button does not redirect to FxA

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/123done#171]
When I click the "Sign in with PKCE" button, I am not redirected to FxA.

Opening the browser console, a network request is made to the content-server's /authorization endpoint. The /authorization endpoint does not have CORS and the request fails.

If enabling CORS on that route, the request then fails because /authorization only returns HTML where this call expects a JSON response.

**Comment by vladikoff** [comment]
Yeah, the button broke once we moved the redirect to the server I think
